lightbulb_bot/__main__.py

```diff
@@ -2,10 +2,17 @@
 
 import hikari
 import lightbulb
-
+from hikari import Intents
 import lightbulb_bot
 
 
+INTENTS = (
+    Intents.ALL_MESSAGES
+    | Intents.MESSAGE_CONTENT
+    | Intents.GUILD_MEMBERS
+    | Intents.GUILDS
+)
```

Comment:
Why are these intents different from the ones in the Crescent bot?

Reply:
Because for prefix commands to work, ALL_MESSAGES and MESSAGE_CONTENT was necessary, but since I'll be removing prefix commands I'll remove those intents too

Reply:
Cool, sounds good.